std_msgs/Header header
rbd_spot_perception/GraphNavWaypoint[] waypoints